default

```diff
@@ -10,6 +10,6 @@ U_BOOT_MENU_LABEL="Vicharak Linux Kernel"
 U_BOOT_PARAMETERS=$(cat /etc/kernel/cmdline)
 #U_BOOT_ROOT=""
 U_BOOT_TIMEOUT="10"
-U_BOOT_FDT="rk3399-vaaman-linux.dtb"
+U_BOOT_FDT="rk-kernel.dtb"
```

Comment:
In case of vaaman, are you sure rk-kernel.dtb is getting linked with rk3399-vaaman-linux.dtb?

Reply:
okay, works